js/solGS/solGS.js

```diff
@@ -566,6 +566,42 @@ solGS.waitPage = function (page, args) {
 	    type    : 'POST',
  	    data    : profile,
 	    url     : '/solgs/run/saved/analysis/',
+	    success : function(response) {
+		if (response.result.match(/Submitted/)) {
+		    confirmRequest();
+		} else {
+		    jQuery('<div />')
+			.html('Error occured submitting the job. Please contact the developers.' + "\n\nHint: " + response.result)
+			.dialog({
```

Comment:
https://www.w3schools.com/bootstrap/bootstrap_modal.asp
The modal is defined in html in a mas file, then you can use jQuery or a html data trigger to open the modal. Then you can use jQuery to run events when buttons are clicked (e.g. the OK button)